pithos/pithos.py

```diff
@@ -1460,22 +1463,63 @@ def restore_position(self):
         if not x is None and not y is None:
             self.move(int(x), int(y))
 
+    def restore_size(self):
+        """ resizes window to size stored in preferences """
+        w, h = self.settings['win-size']
+        if not w is None and not h is None:
```

Comment:
__[Flake8]__

test for object identity should be 'is not'


<!-- Comment by [SideCI](https://sideci.com) -->